.travis.yml

```diff
@@ -27,6 +27,10 @@ before_install:  # Ensure we have the required versions of Python and Pip.
   - 'echo "TRAVIS_PULL_REQUEST_SHA: $TRAVIS_PULL_REQUEST_SHA"'
   - 'echo "TRAVIS_COMMIT: $TRAVIS_COMMIT"'
   - 'git --no-pager log -n2'
+  - 'pwd'
+  - 'ls -la'
```

Comment:
you probably want to remove these

Reply:
No, I deliberately left these in because I'd like to keep an eye on what's getting cached by Travis's cacher.